src/p2p_plugin.c

```diff
@@ -106,7 +106,7 @@ ncclResult_t nccl_p2p_gdr_support(int dev)
   static int module_loaded = -1;
 
   if (module_loaded == -1) {
-#if defined(__HIP_PLATFORM_HCC__) || defined(__HCC__) || defined(__HIPCC__)
+#if defined(__HIP_PLATFORM_AMD__) || defined(__HIPCC__)
```

Comment:
For my understanding, why is this change required?

Reply:
SWDEV-427469